src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -843,6 +819,8 @@ const DEFAULT_VALUES = {
   defaultScore: 0,
   dragMargin: 8,
   selectedDef: null,
+
+  numberOfBins: 32,
```

Comment:
You right, I forget to register the get/set for numberOfBins for all of those.

A way to handle those, is to capture all the component instance that care about the number of bins in an array and implement a method like follow.

```
function setNumberOfBins(size) {
   binAwareArray.forEach(c => c.setNumberOfBins(size));
}
```